src/Compilers/CSharp/Portable/Declarations/MergedTypeDeclaration.cs

```diff
@@ -224,5 +225,33 @@ public ICollection<string> MemberNames
                 return _lazyMemberNames;
             }
         }
+
+        /// <summary>
+        /// This method is an allocation-free implementation of
+        /// <see cref="Symbol.IsDefinedInSourceTree(SyntaxTree, TextSpan?, CancellationToken)"/>. If this method returns
+        /// <see langword="null"/>, the caller should check the containing type instead.
+        /// </summary>
+        internal bool? IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedWithinSpan, CancellationToken cancellationToken)
```

Comment:
Where is this used?

Reply:
It used to be used. I refactored and meant to remove it.

Reply:
➡️ This is now removed (I amended the previous commit since no changes were intended to be submitted for this file).

Reply:
In that case, no need to extract a helper method on `SingleNamespaceOrTypeDeclaration` since there'd only be one caller.